framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -939,6 +939,13 @@ public void typeProcess(TypeElement e, TreePath p) {
         } else {
             previousErrorCompilationUnit = null;
         }
+        if (visitor == null) {
+            // typeProcessingStart invokes initChecker, which should
+            // have set the visitor. If the field is still null, an
+            // exception occured during initialization, which was already
+            // logged there. Don't also cause a NPE here.
+            return;
```

Comment:
The CompletionFailure from the other change leaves the visitor null, which then causes a related NPE here.